pilot/cmd/pilot-agent/main.go

```diff
@@ -476,9 +479,9 @@ func getDNSDomain(domain string) string {
 	return domain
 }
 
-// check if SDS UDS path and token path exist, if both exist, requests key/cert
-// using SDS instead of secret mount.
-func detectSds(controlPlaneBootstrap, controlPlaneAuthEnabled bool, udspath, trustworthyJWTPath string) (bool, string) {
+// detectSds checks config for control plane SDS. The returned values are used by caller
+// to decide whether SDS is used for control plane.
```

Comment:
Since you have moved the controlPlaneAuthEnabled out, can you change the comment to:
detectSds checks if the UDS and JWT paths are present.

Reply:
Done.